app/domain/authentication/authn_jwt/input_validation/parse_claim_aliases.rb

```diff
@@ -71,10 +71,16 @@ def alias_tuple_values(tuple)
             .map(&:strip)
           raise Errors::Authentication::AuthnJwt::ClaimAliasInvalidFormat, tuple unless values.length == 2
 
-          [valid_claim_value(values[0], tuple),
+          [valid_claim_name(values[0], tuple),
            valid_claim_value(values[1], tuple)]
         end
 
+        def valid_claim_name(value, tuple)
+          raise Errors::Authentication::AuthnJwt::ClaimAliasInvalidFormat, tuple if value.blank?
```

Comment:
`valid_claim_value` routine already has this check so I guess that this one can be removed